package-lock.json

```diff
@@ -1,10004 +1,13 @@
 {
     "name": "aws-toolkit-vscode",
     "version": "1.25.0-SNAPSHOT",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
@gsweetwood  I think you need to update npm or node on your machine. This should not be reverted to the old version...

Reply:
You were correct, my npm version was 6.* when it needs to be at lease 7.0 to use lockfile version 2. Installed newer version and ran npm install to create the new package-lock.